js/src/threepio.js

```diff
@@ -1,6 +1,7 @@
 import * as tf from '@tensorflow/tfjs';
```

Comment:
I wonder if it's possible to remove that import.
We could pass `tf` into `Threepio()` constructor.
Also user code can just execute something like `tf[cmd.execFnName](...cmd.args)`.

Reply:
We could! Would this make things easier?

Reply:
@Nolski We need to do this. Good idea @vvmnnnkv

Reply:
Rather than even do that, you could just assume that the `tf` object is a global that will exist. So long as it's being imported for syft.js successfully, threepio can assume it's there. Should be able to do: `/* global tf */` at the top of the file.